lib/spack/env/cc

```diff
@@ -40,6 +40,17 @@ parameters=(
     SPACK_SYSTEM_DIRS
 )
 
+# This is an array of optional parameters that aren't required to be set
+# We don't technically need a list here but for documentation.
+optional_parameters=(
```

Comment:
I like having it for documentation, but spellcheck complains about it being unused.  Maybe make it a comment?

Reply:
Done.